docs/packages/dara-components/plotting/introduction.mdx

```diff
@@ -169,10 +169,27 @@ You should use:
 You can choose a default theme for all your plotting components using the following
 
 ```python
-from dara.components.plotting import set_default_bokeh_theme, set_plotly_default_theme
+from dara.components.plotting import set_default_bokeh_theme, set_default_plotly_theme
 
 # provide theme object as expected by bokeh Document, see docs https://docs.bokeh.org/en/2.4.1/docs/reference/document.html#bokeh.document.Document.theme
 set_default_bokeh_theme({ ... })
 
 set_default_plotly_theme({ ... })
 ```
+
+For example, for Plotly, see the following example:
+
+```python
+import plotly.express as px
+from dara.components import Plotly
+from dara.components.plotting import set_default_plotly_theme, dark_theme
```

Comment:
This is not quite the correct import because there exists a `dark_theme` for both `Plotly` and `Bokeh` and these are different. So I am not sure which one would get prioritised in the import and if this works, something to look into.

So if using plotly to guarantee the correct one should be:
`from dara.components.plotting.plotly import dark_theme`

We can add a note explaining where to get these themes from and that they could use this as a base to tweak and make their own

Reply:
Also you can import `set_default_plotly_theme` from top level with the plotly import if you want

Reply:
Good catch, as for which one is prioritised - the one which is present in `plotting/__init__.py` second would take priority (at the moment it's plotly). It's unfortunate both have the same name, I guess best thing to do is to use the more specific import as you've said

Reply:
done!